src/Coin.cpp

```diff
@@ -6,37 +6,38 @@
 
 #include "Coin.h"
 
+#include "ARK/Address.h"
 #include "Aeternity/Address.h"
 #include "Aion/Address.h"
 #include "Bitcoin/Address.h"
-#include "Bitcoin/SegwitAddress.h"
 #include "Bitcoin/CashAddress.h"
+#include "Bitcoin/SegwitAddress.h"
+#include "Bravo/Address.h"
+#include "Cosmos/Address.h"
 #include "Decred/Address.h"
+#include "EOS/Address.h"
 #include "Ethereum/Address.h"
+#include "FIO/Address.h"
 #include "Groestlcoin/Address.h"
+#include "Harmony-One/Address.h"
 #include "IOST/Account.h"
 #include "Icon/Address.h"
-#include "Nano/Address.h"
+#include "IoTeX/Address.h"
 #include "NEO/Address.h"
+#include "Nano/Address.h"
+#include "Nebulas/Address.h"
 #include "Nimiq/Address.h"
 #include "Ontology/Address.h"
 #include "Ripple/Address.h"
+#include "Semux/Address.h"
+#include "Steem/Address.h"
 #include "Stellar/Address.h"
-#include "Cosmos/Address.h"
 #include "Tezos/Address.h"
 #include "Tron/Address.h"
 #include "Wanchain/Address.h"
+#include "Waves/Address.h"
 #include "Zcash/TAddress.h"
-#include "Bravo/Address.h"
```

Comment:
why did we remove these?

Reply:
Nothing removed, reordered according to clang-format, rest of code base agrees with clang-format output.